lib/plugins/aws/lib/naming.js

```diff
@@ -182,6 +182,15 @@ module.exports = {
   getApiKeyLogicalIdRegex() {
     return /^ApiGatewayApiKey/;
   },
+  getUsagePlanLogicalId() {
+    return 'ApiGatewayUsagePlan';
+  },
+  getUsagePlanKeyLogicalId(usagePlanKeyNumber) {
```

Comment:
The rules for the generation of those logical ids and some examples should be added to the docs (https://github.com/serverless/serverless/blob/master/docs/providers/aws/guide/resources.md#aws-cloudformation-resource-reference).